### PR TITLE
[New Rule] PowerShell Kerberos Ticket Dump

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Detects PowerShell scripts that have the capability of dumping Kerberos tickets from LSA, which potentially indicates an attacker attempt to acquire credentials for lateral movement.

### CODE REVIEW

**Review by Aegrah — APPROVED** [review]
LGTM. Do we also capture this behavior when not using powershell? For example when running a tool like rubeus? https://github.com/GhostPack/Rubeus/blob/master/Rubeus/Commands/Dump.cs Would be an interesting addition if we don't have that already.

File: rules/windows/credential_access_posh_kerb_ticket_dump.toml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects PowerShell scripts that have the capability of dumping kerberos tickets from LSA, which potentially indicates an
+attacker attempt to acquire credentials for lateral movement.
```
Comment (DefSecSentinel) [inline rules/windows/credential_access_posh_kerb_ticket_dump.toml:13]:
```suggestion
Detects PowerShell scripts that have the capability of dumping kerberos tickets from LSA, which potentially indicates an attacker attempt to acquire credentials for lateral movement.
```

**Review by DefSecSentinel — APPROVED** [review]
LGTM

File: rules/windows/credential_access_posh_kerb_ticket_dump.toml (hunk 1 above)
Comment (w0rk3r) [reply rules/windows/credential_access_posh_kerb_ticket_dump.toml]: We break the description at 120 on DRs

File: rules/windows/credential_access_posh_kerb_ticket_dump.toml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+integration = ["windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects PowerShell scripts that have the capability of dumping kerberos tickets from LSA, which potentially indicates an attacker attempt to acquire credentials for lateral movement.
```
Comment (w0rk3r) [inline rules/windows/credential_access_posh_kerb_ticket_dump.toml:12]:
```suggestion
Detects PowerShell scripts that have the capability of dumping Kerberos tickets from LSA, which potentially indicates an
attacker's attempt to acquire credentials for lateral movement.
```

**Pull request merged**